.github/pull_request_template.md

```diff
@@ -6,4 +6,4 @@
 
 - [ ] I have renamed all attached images with descriptive file names (or I did not include any images)
 - [ ] Before creating a new note, I searched the vault (or I only edited existing notes)
-- [ ] I linked any newly created notes in a MoC or overview note.
+- [ ] In case I created a new note in the folder `04 - Guides, Workflows, & Courses`, I added a link to the new note(s) in one of the `for {group X}` overviews ([listed here](https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/%F0%9F%97%82%EF%B8%8F+04+-+Guides%2C+Workflows%2C+%26+Courses)).
```

Comment:
```suggestion
- [ ] (Optional) In case I created a new note in the folder `04 - Guides, Workflows, & Courses`, I added a link to the new note(s) in one of the `for {group X}` overviews ([listed here](https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/%F0%9F%97%82%EF%B8%8F+04+-+Guides%2C+Workflows%2C+%26+Courses)).
```

Reply:
Just added the parenthesis to make it optional